src/components/__tests__/refinement-list.js

```diff
@@ -17,6 +17,8 @@ const addFacet = jest.fn();
 const searchStore = {
   getFacetValues,
   addFacet,
+  stop: () => {},
```

Comment:
can you jest.fn these so you test they get called correctly?

Reply:
just testing they got called add nothing we would need to test the got called in order.